### ISSUE TITLE
Data Preprocessing

### ISSUE THREAD

**Issue by kwangHouzz** [FengyangZhang/caffe_roomnet#6]
Hi Fengyang,

How do you do data preprocessing (like scaling, subtracting mean value) in your model?  It seems that you do not specify any scaling factor or mean value in either roomnet_multistage.prototxt or roomnet_train.prototxt?

Thanks!

**Comment by FengyangZhang** [comment]
Those are in the roomnet_data_layer.cpp I think.

**Comment by kwangHouzz** [comment]
Yes I understand that data transformation is defined in roomnet_data_layer.cpp. But the problem is that **you are not defining any value of scaling factor/ mean in the prototxt.** Was is true that you were not doing any data preprocessing in your training/testing? Please correct me if I am wrong.

Thanks!

**Comment by kwangHouzz** [comment]
BTW, what accuracy does your type classifier obtain in your experiment? I tried to train your model, and the validation accuracy was not even close to that of the original paper (81.5%).
